src/native/minipal/time.c

```diff
@@ -107,7 +107,7 @@ void minipal_microdelay(uint32_t usecs, uint32_t* usecsSinceYield)
 
         if (usecsSinceYield)
         {
-            usecsSinceYield = 0;
+            *usecsSinceYield = 0;
```

Comment:
cc @VSadov Bug introduced in https://github.com/dotnet/runtime/pull/103212/files#diff-c176d2f20b00577173705c18748a4d5fde971980736f8c785e2964482f5ea3dcR87

Can fixing this bug throw-off the perf tunning that was done in that PR?

Reply:
Unlikely to affect perf. If we start calling `Sleep(1)` we are already in some kind of trouble. It is basically a panic mode. 

On windows it will take at least 1-16 msec. and we just hope it may help. Maybe somehow our spinning starves what we are waiting for. (which is not very likely these days as everything runs multiple cores, but we do not have a lot of other options).

Reply:
On Linux nanosleep is more precise, but overal the effect of the bug is that after spinning for 10usec we would sleep on every call, instead of every 10usec. 

Either way in "fast" cases we do not expect to sleep much or at all.